redbox/models/chat.py

```diff
@@ -35,11 +36,32 @@ class ChatRequest(BaseModel):
     }
 
 
-class ChatResponse(BaseModel):
-    response_message: ChatMessage = Field(description="The response message")
+class Metadata(BaseModel):
+    parent_doc_uuid: UUID = Field(description="uuid of original file")
+    languages: Optional[list[str]] = Field(
+        description="languages detected in this chunk", examples=[["eng"]], default=None
+    )
+    url: Optional[AnyUrl] = Field(description="url of original file", default=None)
+    filetype: Optional[str] = Field(description="content-type", examples=["application/pdf"], default=None)
+    page_numbers: Optional[list[int]] = Field(
+        description="page number of the file that this chunk came from", default=None
+    )
 
-    model_config = {
-        "json_schema_extra": {
-            "examples": [{"response_message": {"text": "AI stands for Artificial Intelligence.", "role": "ai"}}]
-        }
-    }
+
+class InputDocuments(BaseModel):
+    page_content: str = Field(description="chunk text")
+    metadata: Metadata
+
+
+class ChatResponse(BaseModel):
+    question: str = Field(
+        description="original question",
+        examples=["Who is the prime minister?"],
+    )
```

Comment:
the frontend know all of the documents that could have been used in RAG, but not the documents actually used in RAG. The point of this change is that the document references come back in a helpful format rather than the `Doc12AC-0FE-8CB4-....` text format

Reply:
ahh, good point. yes lets drop it, its unhelpful